.gitignore

```diff
@@ -1,92 +1,2 @@
-build/
-bin/
-lib/
-Debug/
-PreRelease/
-Profile/
-Release/
-ipch/
-__pycache__/
-src/data
-debian/autoreconf*
-debian/files
-debian/*.debhelper.log
-debian/*.substvars
-debian/pioneer*/
-autom4te.cache/
-codedoc/
-nd/Data/
-doxygen/html
-doxygen/latex
-*.vcxproj.user
-*.user
-*.ncb
-*.suo
-*.sdf
-*.opensdf
-*.o
-*.a
-.deps
-/*.exe
-/*.dll
-/*.pdb
-/*.ilk
-Makefile
-Makefile.in
-.DS_Store
-*.swp
-shipstat.csv
-pioneer
-pioneer.exe
-modelviewer
-modelviewer.exe
-modelcompiler
-modelcompiler.exe
-modelcompiler-debug.lib
-modelcompiler-prerelease.lib
-modelcompiler-profile.lib
-modelcompiler.lib
-pioneer-debug.lib
-pioneer-prerelease.lib
-pioneer-profile.lib
-pioneer.lib
-pioneer-x64*.lib
-aclocal.m4
-config.guess
-config.log
-config.status
-config.sub
-configure
-depcomp
-install-sh
-missing
-compile
-test-driver
-stamp-h1
-*~
-*.orig
-pioneer-*.tar.*
-Thumbs.db
-*.wwdb
-gmon.out
-.project
-.cproject
-.dirstamp
-result
 *.sgm
-.vs/
-*.user
-*.exp
-*.VC.db
-/release
-# Core dumps shouldn't be tracked by git.
-.[0-9][0-9]*
-# Don't track C++ language server cache files.
-.ccls-cache
-.clangd/
-.cache/
```

Comment:
Why don't we want to ignore cache?

Reply:
I admit, although it seems to me that this is a personal workflow - neither `ccls` nor `LuaLS` leave anything in my working tree.